test/specs/print_utils/tests/01.print_utils.js

```diff
@@ -61,6 +61,26 @@ exports.execute = function (options) {
             expect(printText({key:123})).toBe('{"key":123}');
             expect(printText({key:123,subkey:{subsubkey:456}})).toBe('{"key":123,"subkey":{"subsubkey":456}}');
         });
+        
+        it('printJSON() should format JSON values correctly for Edit Context.', function() {
+            var printJSON = formatUtils.printJSON;
+            var options={context:'entry/edit'};
+            expect(printJSON(null,options)).toBe('null');
+            expect(printJSON('',options)).toBe('""');
+            expect(printJSON(true,options)).toBe('true');
+            expect(printJSON(false,options)).toBe('false');
+            expect(printJSON(2.9,options)).toBe('2.9');
+        });
+        
+        it('printJSON() should format JSON values correctly for Not Edit Context.', function() {
```

Comment:
Also add a JSON object, to see that it's actually returning the beautified version.